pkg/proxy/userspace/proxier.go

```diff
@@ -167,6 +167,7 @@ type Proxier struct {
 	serviceChangesLock sync.Mutex
 	serviceChanges     map[types.NamespacedName]*serviceChange // map of service changes
 	syncRunner         asyncRunnerInterface                    // governs calls to syncProxyRules
+	forceReload        bool
```

Comment:
the userspace parts of this patch are all already in sdn's copy. (And the remaining iptables proxier bit can be merged into `UPSTREAM: <carry>: Allow overriding/hooking into kube-proxy`)